packages/ui/src/components/templates/report/components/AdsAndSocialMedia/AdsAndSocialMedia.tsx

```diff
@@ -1,104 +1,236 @@
+                        ),
+                      )}
+                    </div>
+                  </div>
+
+                  <a
+                    className={buttonVariants({
+                      variant: 'link',
+                      className:
+                        'h-[unset] cursor-pointer !p-0 !text-[#14203D] underline decoration-[1.5px]',
                     })}
+                    href={link}
                   >
-                    {!!items?.length &&
-                      items.map(({ label, value }) => {
-                        return (
-                          <li key={label} className={'list-disc'}>
-                            <TextWithNAFallback className={'me-2 font-semibold'}>
-                              {toTitleCase(label ?? '')}:
-                            </TextWithNAFallback>
-                            <TextWithNAFallback as={AnchorIfUrl} className={'break-all'}>
-                              {value}
-                            </TextWithNAFallback>
-                          </li>
-                        );
-                      })}
-                    {!items?.length && <li>No social media presence detected.</li>}
-                  </ul>
+                    <Image
+                      key={src}
+                      src={src}
+                      alt={`${capitalize(provider)} image`}
+                      role="link"
+                      className="h-auto max-h-96 w-auto"
+                    />
+                  </a>
                 </div>
-              ))}
-            {!mediaPresence?.length && (
-              <div>
-                <ul>
-                  <li>No social media presence detected.</li>
-                </ul>
-              </div>
-            )}
-          </div>
-          <div className={'grid grid-cols-[400px_400px] gap-8'}>
-            {adsImages.map(({ provider, src, link }, index) => (
-              <AdImageWithLink
-                title={`${valueOrNA(toTitleCase(provider ?? ''))} Image`}
-                key={src}
-                src={src}
-                alt={`${provider} ad ${index + 1}`}
-                link={link}
-              />
-            ))}
+              ) : (
+                <div className="my-4 flex items-center gap-2 text-gray-400">
+                  <BanIcon className="h-5 w-5" />
+                  <span className="text-sm">No {capitalize(provider)} profile detected.</span>
+                </div>
+              )}
+            </Card>
+          );
+        })}
+      </div>
+    </div>
+
+    {/* TODO: Shows failing state by default for now */}
+    <div>
+      <h3 className="mb-2 text-base font-bold">Ads</h3>
+      <Card
+        className={ctw(
+          'flex w-full justify-between p-4 shadow-lg',
+          // relatedAdsImages && relatedAdsImages.length > 0 ? 'opacity-100' : 'opacity-60',
+          false ? 'opacity-100' : 'opacity-60',
+        )}
+      >
+        {/* relatedAdsImages */}
+        {false ? (
```

Comment:
Can you explain this part?

Reply:
i left it hidden by default using hardcoded condition, since there was no logic for ads. as Matan mentioned it can be removed, I replaced harcoded parts with the actual logic and commented out the entire ads section.